cernopendata/config.py

```diff
@@ -64,9 +64,32 @@
         template='invenio_marc21/detail.html',
         permission_factory_imp=None,
     ),
+    glossid=dict(
```

Comment:
A "glossary" is a "list of terms", so using "glossid" (for "glossary ID") may be a bit misleading here.  We have only one glossary composed of many terms, and we need to assign IDs not to glossaries, but to these terms. We want to express the notion of a "glossary term ID".